n2ws/manifest.json

```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
```

Comment:
The keys of this json cannot be changed once the integration tile has been deployed (as is the case). However this is not visible in app. If you want to change the title you can do it on the dashboard json itself.